BetterShardsBukkit/src/main/java/vg/civcraft/mc/bettershards/listeners/BetterShardsListener.java

```diff
@@ -370,25 +375,25 @@ public void playerSleepInBed(PlayerInteractEvent event) {
 	
 	// Start of methods to try and stop interaction when transferring.
 	
-	@EventHandler(priority = EventPriority.HIGHEST)
+	@EventHandler(priority = EventPriority.LOWEST)
 	public void playerDropItem(PlayerDropItemEvent event) {
 		Player player = event.getPlayer();
-		if(plugin.isPlayerInTransit(player.getUniqueId())) {
+		if(BetterShardsPlugin.getTransitManager().isPlayerInTransit(player.getUniqueId())) {
 			event.setCancelled(true);
 		}
 	}
 	
-	@EventHandler(priority = EventPriority.HIGHEST)
+	@EventHandler(priority = EventPriority.LOWEST)
 	public void playerDamageEvent(EntityDamageEvent event) {
-		if (!(event.getEntity() instanceof Player))
+		if (!(event.getEntity().getType() == EntityType.PLAYER))
```

Comment:
Well but anything with Player as EntityType will also be a player. Is this just a compiler warning thing? How bad is using instanceof actually?